src/cellmanager.js

```diff
@@ -554,7 +563,18 @@ export default class CellManager {
                 colIndex,
                 rowIndex
             } = $.data($cell1);
+            if (rowIndex === undefined) {
+                // this must be a footer
+                const {isTotalRow} = $.data($cell1);
```

Comment:
Instead of checking if rowIndex is undefined, why not extract isToTalRow in the first place on line 565 and check

Reply:
Also a special case for total row doesn't make sense here. Instead just grab the value of the cell and call copyToClipboard once